regress/expected/repack_2.out

```diff
@@ -338,25 +352,19 @@ CREATE FUNCTION trgtest() RETURNS trigger AS
 $$BEGIN RETURN NEW; END$$
 LANGUAGE plpgsql;
 CREATE TABLE trg1 (id integer PRIMARY KEY);
-CREATE TRIGGER z_repack_triggeq BEFORE UPDATE ON trg1 FOR EACH ROW EXECUTE PROCEDURE trgtest();
+CREATE TRIGGER repack_trigger_1 AFTER UPDATE ON trg1 FOR EACH ROW EXECUTE PROCEDURE trgtest();
```

Comment:
I think these are spurious changed caused by some cross-pollination after your MR #107 (I thought it was already merged to master). I've made a mistake myself because yesterday, I committed ac1333f9 thinking I forgot it but actually the doc change was in a branch I haven't merged yet.

I think we should merge some of our half finished things into master...